bin/ruby-build

```diff
@@ -968,11 +968,13 @@ build_package_verify_openssl() {
 }
 
 # Ensure that directories listed in LDFLAGS exist
+# $LDFLAGS must be in format '-L /path/' or '-L/path/'
 build_package_ldflags_dirs() {
   local arg
   for arg in $LDFLAGS; do
     case "$arg" in
-    -L* ) mkdir -p "${arg#-L}" ;;
+    -L ) ;;
+    * ) mkdir -p "${arg#-L}" ;;
```

Comment:
Handling the `*` argument (which matches any string) is not optimal here because LDFLAGS might have arguments that are different from `-L` and directory names. I've pushed a commit that fixes the original issue, but ignoring any potential arguments that might not be related to `-L`.